### PR TITLE
[INFRA] Add release mail templates

### PR DESCRIPTION
**Pull request by pan3793**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
will they go to the source release？

**Comment by pan3793** [comment]
Yeah, sounds like we should add license header or exlude them.

File: build/release/tmpl/announce.tmpl [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Title: [ANNOUNCE] Release Apache Kyuubi(Incubating) {release_version}
+
+Content:
+Hi all,
+
+The Apache Kyuubi(Incubating) community is pleased to announce
+that Apache Kyuubi(Incubating) {release_version} has been released!
+
+Apache Kyuubi is a one-stop data streaming platform that provides automatic, secure,
```
Comment (yaooqinn) [inline build/release/tmpl/announce.tmpl:9]: do we have such thing？

Comment (pan3793) [reply build/release/tmpl/announce.tmpl]: Removed

File: build/release/tmpl/general_incubator_vote_result.tmpl [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+(* = binding)
++1:
+- {AA}*
+- {BB}
+- {CC}*
+- {DD}*
+
++0: None
+
```
Comment (timothy65535) [inline build/release/tmpl/general_incubator_vote_result.tmpl:35]: seems we can remove the empty line

File: build/release/tmpl/dev_kyuubi_vote_result.tmpl [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+(* = binding)
++1:
+- {AA}*
+- {BB}
+- {CC}*
+- {DD}*
+
++0: None
+
```
Comment (timothy65535) [inline build/release/tmpl/dev_kyuubi_vote_result.tmpl:33]: seems we can remove the empty line

File: build/release/tmpl/general_incubator_vote_result.tmpl (hunk 2 above)
Comment (pan3793) [reply build/release/tmpl/general_incubator_vote_result.tmpl:35]: Keep it consistent, see https://www.mail-archive.com/dev@spark.apache.org/msg27936.html

**Review by yaooqinn — APPROVED** [review]

**Pull request closed without merging**